artemis-protocols/artemis-proton-plug/src/main/java/org/proton/plug/context/server/ProtonServerReceiverContext.java

```diff
@@ -116,7 +116,10 @@ public void onMessage(Delivery delivery) throws ActiveMQAMQPException {
          receiver = ((Receiver) delivery.getLink());
 
          if (!delivery.isReadable()) {
```

Comment:
This was supopsed to be  logger.trace()... can you do a logger.trace instead of removing it?